<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>International Fabry Disease Genotype-Phenotype Database (dbFGP)</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="../public/css/style.css">
	<link rel="stylesheet" type="text/css" href="./css/styles_search.css" />
</head>
<style>

</style>
<body>
<div class="container">
<?php include"header.php"; ?>
	<div class="content">
		<h2 style="font-family: 'Helvetica Neue', sans-serif; font-size: 18px; color:#3D1C00;font-weight: 700; line-height: 50px; letter-spacing: 1px;">
			International Fabry Disease Genotype-Phenotype Database(dbFGP)
		</h2>

	<P style="text-align: left; background-color:#fff;color: #292929; font-style: italic; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px;  border: 1px solid #fff">
		The International Fabry Disease Genotype-Phenotype Database (dbFGP), is a Fabry disease-specific web-based database that provides information on the genotypes and phenotypes of gene
		mutations that cause the two major clinical subtypes of Fabry disease. The database consolidates information from peer-reviewed publications, other databases,
		including the Human Gene Mutation Database (HGMD) and The Japanese Fabry Database (fabry-database.org), and patient clinical and biochemical data from the diagnostic and clinical
		evaluations of patients diagnosed and evaluated at the Mount Sinai International Center for Fabry Disease (ICFD),
		as well as expert Fabry disease diagnostic and treatment centers worldwide.
	</p>
	<P style="text-align: left; color: #292929; font-style: italic; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px;  border: 1px solid #fff">
		The dbFGP is the effort of a team of physicians, genetic counselors, researchers, and bioinformatics specialists from the Department of Genetics and Genomic Sciences at the Icahn School of Medicine at Mount Sinai in New York City,
		as well as an international group of collaborators with expertise in Fabry disease</p>
	<P style="text-align: left; font-style: italic; color: #292929; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px; border: 1px solid #fff">

		The purpose of this database is to provide healthcare providers, patients, and their families with easily accessible and comprehensive information about Fabry disease, genotype/phenotype correlations, and assistance in the diagnosis and medical management of patients and family members with specific mutations in the &alpha;-galactosidase A gene (GLA) which cause Fabry disease. We fully appreciate that the clinical expression and severity of the disease is related to the absence or enzymatic residual
		activity of &alpha;-galactosidase A (&alpha;-Gal A), and that the clinical spectrum among individuals with residual enzyme activity is a continuum. However, there are two major clinical subtypes, the Type 1 Classic and the Type 2 Later-Onset phenotypes. Determination of which phenotype results from a given mutation is important for medical management, genetic counseling, and decisions relevant when to initiate treatment in affected males and heterozygous females. </p>
	<P style="text-align: left; font-style: italic; color: #292929; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px; border: 1px solid #fff">

		The clinical, biochemical, and genetic information used to assign the major phenotypes for each Fabry mutation is based on review of clinical evaluations of patients with specific mutations, review of the literature, and, for missense mutations, laboratory studies to determine the relative residual activity of the mutant &alpha;-Gal A enzyme.
	</p>
	<P style="text-align: left; font-style: italic; color: #292929; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px; border: 1px solid #fff">
	It should be noted that the Fabry Genotype-Phenotype Database has been established and is maintained with funds from the Department of Genetics and Genomic Sciences, and without support from Industry. The dbFGP will continue to be maintained and updated by clinical and bioinformatics specialists in the Department of Genetics and Genomic Sciences at the Icahn School of Medicine at Mount Sinai, New York, NY.
		</p>
	<P style="text-align: left; font-style: italic; background-color:#fff;color: #292929; display: inline-block; font-family: 'Georgia', serif; font-size: 16px; font-weight: 500; line-height: 22px; margin: 0 90px 4px 90px; padding: 1px 10px 1px 10px;border-bottom: 0px solid #69D2E7;">
	The dbFGP invites contributions of Fabry disease genotype-phenotype information from physicians, researchers, and diagnosticians. To submit data for the database, please access the “Data Contributions” tab above.
	</p>

	<p style="text-align: center;font-style: italic;font-family: 'Helvetica Neue', sans-serif; font-size: 16px; color:#3D1C00;font-weight: 600; line-height: 50px; letter-spacing: 0.5px; float:center">
		For each GLA mutation, the following information, if available, is presented:
		<ul style="list-style-type: square;text-align:left;background-color:#fff;color:#292929; margin-top:0px;display: inline-block; font-family: 'Georgia', serif; font-size: 16px; line-height: 22px;font-weight:500">
			<li style="margin-left: 105px;">Comprehensive genotype information
				<ul style="list-style-type: circle;padding: 2px;">
					</li><li style="margin-left: 15px;">Codon change
					</li><li style="margin-left: 15px;">Amino acid change
					</li><li style="margin-left: 15px;">Standardized nomenclature (cDNA, protein, gDNA)
					</li><li style="margin-left: 15px;">Exon(s) / Intron(s) involved
					</li><li style="margin-left: 15px;">Mutation type
					</li><li style="margin-left: 15px;">HGMD accession number
			</li></ul>
			<li style=" margin-left: 105px;">Likely phenotype
			</li><li style=" margin-left: 105px;">Hyperlinked references
			</li><li style=" margin-left: 105px;">Clinical information of affected males (based on peer-reviewed publications, Mount Sinai’s Clinical Database, Data Contributions)
			</li><li style=" margin-left: 105px;">Phenotype-specific clinical picture, variation or other considerations
			</li><li style=" margin-left: 105px;">Phenotype-specific recommendations for treatment, medical management, and testing of at-risk family members
			</li>
		</ul>
	</p>
		<P style="text-align: left; background-color:#fff;color: #26ADE4; display: inline-block; font-family: 'Georgia', serif; font-size: 14px; font-style: italic; font-size: 12px; line-height: 22px;font-weight:500; margin: 5px 90px 4px 90px; padding: 10px 10px 10px 10px;border-bottom: 0px solid #69D2E7;border-top: 1px solid #69D2E7;">
		<b>Disclaimer: </b>Information presented in this database is based on available data and expert interpretation of these data.
		Although all efforts to maintain accurate information and interpretation of these data have been made,
		variation may occur which should be documented and shared as some genotypes may be intermediate with some family members (particularly males)
		having Type 1 Classic manifestations while others may have Type 2 Later-Onset manifestations. In addition, within a family with a particular mutation,
		there may be variation in the presence or absence and severity of symptoms in affected males.
		The variability in heterozygous females is directly related to the percentage of cells expressing the GLA mutation in accord with the
		biologic process termed “random X-chromosomal inactivation”. It is likely that new information will become available to change or modify the presented information. If a user of this database identifies such information, please contact us so we can incorporate the information into the Database.
	</p><br>
	</div>
	<?php include"footer.php"; ?>
</div>
</body>
</html>
